Radiant/fileHandler.js

```diff
@@ -45,40 +45,39 @@ module.exports.watcher = (ouPath, args) => {
           path,
           conversationTopicId: args.conversationTopicId,
           authToken,
-      });
+      }, 0);
     });
 };
 
 module.exports.end = (streamPath) => {
   // check directory for files left
-    setTimeout(() => {
-        watcher.close();
-        fs.readdir(`./media${streamPath}`, (err, files) => {
-            if(err){
-                console.log('-=*[ERROR: no such file or directory ]*=-');
-            }
-            // console.log('-=*[ Cleanup remaining files ]*=-');
-            files.forEach(file => {
-                const fileC = file.split('.')[1];
-                if(fileC === 'm3u8') {
-                    // checkM3U8(`media${streamPath}/${file}`);
-                } else if(fileC === 'DS_Store'){
-                    fs.stat(file, (err) => {
-                        if(err === null) {
-                            fs.unlink(file, (err, data) => {
-                                if(err){
-                                    console.log(`ERROR: File Not Found ${err.message}`);
-                                }
-                            });
-                        } else {
-                            console.log(`File not found ${err}`);
-                        }
-                    });
-                }
-            });
-        });
-    }, 1500);
-
+  //   setTimeout(() => {
```

Comment:
can this be removed completely?

Reply:
yes i removed that and the function it was in